src/server.cc

```diff
@@ -101,6 +101,18 @@ bool Server::getServerInfo(const char* file_name, ServerInfo* info) {
       handler->Init(name, *handler_config);
       info->uri_prefix_to_handler[name] = handler;
 
+    } else if (key == "default") {
+      if(config.statements_[i]->tokens_.size() != 2
```

Comment:
can you make sure to leave a space between all `if`s/`for`s and the open parenthesis